QuasistaticBrownianThermalNoise/QuasistaticBrownianThermalNoise.cpp

```diff
@@ -234,6 +235,11 @@ class ElasticProblem {
   double lossPhi_Iso_FusedSilica, lossPhi_Iso_Ta2O5, lossPhi_AlGaAs,
       lossPhi_Iso_AlGaAs;
 
+  // Loss Angles for Young's tensors that can be varied
+  double phi11, phi12, phi44, lossPhi_Ta2O5_bulk, lossPhi_Ta2O5_shear,
```

Comment:
Rename phi11, etc. to be for AlGaAs, eg lossPhi_AlGaAs_12, etc., here and throughout